extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -85,10 +85,42 @@
   background-color: rgba(0,0,0, 1) !important;
 }
 
+.-amp-lightbox-viewer-mask {
+  z-index: 2147483643 !important;
+}
+
+.-amp-lightbox-viewer-gallery {
+  display: none !important;
```

Comment:
remove !important  from display:none (since we do change this with other CSS, it is not really a constant)